bHY6yh-de2065/cod-yyyiou_bHY6yh-de2065.xml

```diff
@@ -0,0 +1,210 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 8</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+               <name ref="alasdairwsliter">Alasdair W. Sliter</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 8</title>
+               <date when="2023-05-31">May 31, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="C" n="cod-yyyiou">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-05-31" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#V" n="3-r"/>
+            <cb ed="#V" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de2065"><!-- p1l1-r8 -->
+            <head xml:id="bHY6yh-de2065-Hd1e108">Rubrica 8</head>
+            <head xml:id="bHY6yh-de2065-Hd1e111" type="question-title">De iustitia et iudicio Dei</head>
+            <p xml:id="bHY6yh-de2065-d1e114">De charitate seu dilectione et odiositate dei ad hominem. Rubrica. 8.
+    </p>
+            <p xml:id="bHY6yh-de2065-d1e121">
+                <ref xml:id="bHY6yh-de2065-Rd1e118">IOb ystoria capitulo XVIo</ref> <quote xml:id="bHY6yh-de2065-Qd1e125">
```

Comment:
If your references begin on a separate line, they too should start with a line break. This is true for all of the refs here.

Reply:
im not sure what you mean, do you mean putting a line break in the code to represent the beginning of a new reference instead of just where the text breaks?